docs/enterprise-solutions/configuration/infrastructure-configuration/control-other-cline-features/yolo-mode.mdx

```diff
@@ -0,0 +1,82 @@
+---
+title: Yolo Mode
+description: Configure Yolo Mode settings for enterprise environments
+---
+
+## Overview
+
+Yolo Mode enables Cline to operate with increased automation while maintaining enterprise security and compliance standards. When enabled, Cline can execute approved operations with minimal user confirmation, accelerating development workflows.
+
+## Configuration
+
+### Enabling Yolo Mode
+
+Yolo Mode can be configured at the organization level through your enterprise dashboard:
+
+1. Navigate to your enterprise configuration settings
+2. Access the "Advanced Features" section  
+3. Toggle Yolo Mode for your organization
+4. Configure specific automation rules and permissions
```

Comment:
Let;s follow the full cycle here 

1. settings
2. Enterprise settings
3. Features
4. toggle yolo mode 

I don't know what configure specific rules mean here. Would recommend removing

<img width="1200" height="793" alt="Image" src="https://github.com/user-attachments/assets/4f5d4d8b-6169-4376-8dac-2d1fe867a35f" />